### PR TITLE
deps: upgrade LatestVim to be v8.2.3249

### PR DESCRIPTION
**Pull request by myitcv**

### CODE REVIEW

File: _scripts/publishDockerImages.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 source "${BASH_SOURCE%/*}/common.bash"
 
-docker push govim/govim
+docker push -a govim/govim
```
Comment (leitzler) [inline _scripts/publishDockerImages.sh:5]: What's the rational behind this change?

File: testdriver/testdriver.go [hunk 2]
```diff
@@ -887,6 +887,17 @@ func Condition(cond string) (bool, error) {
 	if !semver.IsValid(v) {
 		return false, fmt.Errorf("%v is not a valid semver version", v)
 	}
+	// We have version information. Be perhaps a bit overly
```
Comment (leitzler) [inline testdriver/testdriver.go:890]: Nice change. Maybe we can phrase it a bit different to differentiate `vim` and `vim:v8.2.x`? I think that a future me might be confused when reading tests.

I don't have a nice suggestion on how to improve though. `vimver:v8.2.xxxx` for version?
Or `flavor:vim` / `flavor:gvim`?

Something that differentiate version `vim` from flavour `vim` would be nice.

File: _scripts/publishDockerImages.sh (hunk 1 above)
Comment (myitcv) [reply _scripts/publishDockerImages.sh:5]: Apparently a change in behaviour of `cmd/docker`... at least it appears that way to me. I didn't dig too far, just fixed things up :)

File: testdriver/testdriver.go (hunk 2 above)
Comment (myitcv) [reply testdriver/testdriver.go]: Thanks. It wasn't sitting quite right with me either. So I simplified things even further in https://github.com/govim/govim/pull/1098. Please let me know what you think. The tests hopefully make clear what's going on now too.

**Review by leitzler — APPROVED** [review]
LGTM

**Pull request merged**